vulcan/lib/client/jsx/components/dashboard/card.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import Icon from 'etna-js/components/icon';
+
+import {workflowName} from '../../utils/workflow';
+import ImageMemo from './image_memo';
+
+// To get webpack to pick up the files.
+require('../../../img/umap-thumbnail.png');
```

Comment:
Note, the images are basically stubbed in right now. Once we figure out how we want to do actual workflow metadata and editing, I would imagine the images become served up by Apache or some other means.